foundation/foundation-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -64,6 +64,29 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
+        <!-- START: workaround to get maven build cache invalidation on new SNAPSHOTS of commonly updated plugins:  -->
```

Comment:
A/Q: Since this is in the build-parent, I'd assume downstream projects would pick it up naturally.  Is this not this case and/or is this here as an instructional breadcrumb?